data/data/aws/cluster/vpc/vpc-public.tf

```diff
@@ -52,7 +52,8 @@ resource "aws_subnet" "public_subnet" {
 
   tags = merge(
     {
-      "Name" = "${var.cluster_id}-public-${var.availability_zones[count.index]}"
+      "Name"                   = "${var.cluster_id}-public-${var.availability_zones[count.index]}"
+      "kubernetes.io/role/elb" = ""
```

Comment:
minor nit: Feels odd if this is not set to `shared` since that's the value mentioned below.

Reply:
I was thinking about this myself. Similar tag for private VPCs are created without a set value : https://github.com/openshift/installer/blob/master/data/data/aws/cluster/vpc/vpc-private.tf#L39. But, the comment https://bugzilla.redhat.com/show_bug.cgi?id=2094923#c1 in the BZ, is talking about adding a value "shared" for pre-existing shared VPCs vs none for new ones. Maybe @patrickdillon can provide some context.

Reply:
> a value "shared" for pre-existing shared VPCs vs none for new ones

+1 this is a new vpc so it doesn't need a shared tag

Reply:
We could add an "owned" value just to indicate that it is owned by this cluster because it was created by the cluster. But, "owned"/" " results in the same behavior with the current status of the code.

Reply:
Keeping it consistent with https://github.com/openshift/installer/blob/master/data/data/aws/cluster/vpc/vpc-private.tf#L39, leaving the value empty.